cypress/support/commands.ts

```diff
@@ -14,23 +14,44 @@ import { Result } from "axe-core";
 import "@cypress/code-coverage/support";
 import "cypress-axe";
 
+const loginWithRetry = (iteration: number = 0): void => {
+    if (iteration >= 5) {
```

Comment:
Why 5?

Reply:
It was arbitrary, I just wanted it to happen within 10 seconds although 0.5 + 1 + 2 + 4 + 8 > 10 so I did my maths wrong ;-;